### ISSUE TITLE
JS error in creditCardFrame.js

### ISSUE THREAD

**Issue by Fahl-Design** [heidelpay/php-api#24]
Hi, I found an error in your example JS script. This will fail in IE11 and some other browsers.

Please add some kind of selenium testing to prevent this kind of bugs

https://github.com/heidelpay/php-api/blob/master/example/js/creditCardFrame.js#L21

``` 
var paymentFrameIframe = document.getElementById('paymentIframe');

/**
 * Get hostname and protocol from paymentIframe
 */
targetOrigin = getDomainFromUrl(paymentIframe.src);
``` 
fix: 
``` targetOrigin = getDomainFromUrl(paymentFrameIframe.src); ``` 

best regards

**Comment by heidelpay** [comment]
Hi,

thanks for your response, we will fix this issue.

regards

**Comment by heidelpay** [comment]
Fixed in #25 

Thx to @Fahl-Design
